### DISCUSSION TITLE
Windows Support?

### DISCUSSION THREAD

**Discussion by carloshgamboa** [filecoin-project/lotus#5523] [Storage Provider] [answered]
Hi, I wonder if there are plans to implement a lotus miner for Windows.
Best regards!

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
i don't think so. 

rough, inaccurate quote from some time in the past 3 month "i do not think anyone on the dev team owns a windows computer"

**Comment by magik6k** [comment, marked as answer]
There are no plans for this right now, but all contributions are welcome.

(https://github.com/filecoin-project/lotus/issues/2133 is a related issue, but it's more about the chain node support, which should be simpler)

**Comment by beck-8** [comment]
windows use wsl2
